lib/hsa/hsaBeamformTranspose.cpp

```diff
@@ -19,14 +20,15 @@ hsaBeamformTranspose::hsaBeamformTranspose(Config& config, const std::string& un
                                            bufferContainer& host_buffers,
                                            hsaDeviceInterface& device) :
     hsaCommand(config, unique_name, host_buffers, device, "transpose" KERNEL_EXT,
-               "transpose.hsaco") {
+               "frb_transpose_amd.hsaco") {
+//               "transpose.hsaco") {
     command_type = gpuCommandType::KERNEL;
 
     _num_elements = config.get<int32_t>(unique_name, "num_elements");
     _samples_per_data_set = config.get<int32_t>(unique_name, "samples_per_data_set");
 
     beamform_frame_len = _num_elements * _samples_per_data_set * 2 * sizeof(float);
-    output_frame_len = _num_elements * (_samples_per_data_set + 32) * 2 * sizeof(float);
+    output_frame_len = _num_elements * (_samples_per_data_set + 64) * 2 * sizeof(float);
```

Comment:
Comment where the `64` comes from?

Reply:
Right, worth including a comment directly in the code -- the 64 (like the prior 32) is there to avoid bank conflicts in the transpose output. The 64 just works a hair more efficiently.